examples/jetstream/pull-consumer/java/Main.java

```diff
@@ -0,0 +1,174 @@
+package example;
+
+import io.nats.client.*;
+import io.nats.client.api.ConsumerConfiguration;
+import io.nats.client.api.StreamConfiguration;
+import io.nats.client.api.StreamInfo;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.concurrent.CountDownLatch;
+
+public class Main {
+    public static void main(String[] args) {
+        String natsURL = System.getenv("NATS_URL");
+        if (natsURL == null) {
+            natsURL = "nats://127.0.0.1:4222";
+        }
+
+        // Initialize a connection to the server. The connection is AutoCloseable
+        // on exit.
+        try (Connection nc = Nats.connect(natsURL)) {
+
+            // Access `JetStream` and `JetStreamManagement` which provide methods to create
+            // streams and consumers as well as convenience methods for publishing
+            // to streams and consuming messages from the streams.
+            JetStream js = nc.jetStream();
+            JetStreamManagement jsm = nc.jetStreamManagement();
+
+            // Declare a simple [limits-based stream](/examples/jetstream/limits-stream/java/).
```

Comment:
Same comment as push - remove limits based note. I'm going to have a talk with Byron about this idea and probably go clean up other examples.

Reply:
I confirmed with Byron.

Reply:
Changed the comment to be just:
```
// Declare a simple stream and populate it with a few messages.
```